src/jank/index.js

```diff
@@ -96,4 +104,16 @@ window.onload = () => {
       }
     );
   };
+
+  /**
+   * Redirects the user to the specified scorepad.
+   *
+   * @param {Object} scorepad the scorepad to redirect the user to
+   */
+  function redirectToScorepad(scorepad, sideName) {
+    window.location.href = `${sideName}?scorepad=${scorepad._id}`;
+  }
+  
+  scorepadList = document.getElementById('scorepad-list');
```

Comment:
Hier ginge das wie oben gesagt auch mit lokaler Variable:
```suggestion
  const scorepadList = document.getElementById('scorepad-list');
```